lib/hanami/assets/configuration.rb

```diff
@@ -378,6 +396,25 @@ def asset_url(source)
         "#{ @base_url }#{ compile_path(source) }"
       end
 
+      # An array of digest algorithms to use for generating asset subresource
+      # integrity (SRI) checks
+      #
+      # @since 0.3.0-add-options-to-javascript-helper
+      def sri_algorithm
+        [DEFAULT_SRI_ALGORITHM]
+      end
+
+      # Subresource integrity attribute
+      # @since 0.3.0-add-options-to-javascript-helper
+      # @api private
+      def sri_value(source)
+        if sri
+          result = prefix.join(source)
+          result = digest_manifest.resolve(result)
+          result.fetch('sri').to_s
```

Comment:
@accuser because we can have more than one SRI values, this should do `result.fetch('sri').join(" ")`.